### PR TITLE
[SOL-1604] Fix flaky test

### PR DESCRIPTION
**Pull request by marjev**

### CODE REVIEW

**Comment by marjev** [comment]
JIRA ticket: https://openedx.atlassian.net/browse/SOL-1604

@rlucioni , @mjfrey This should fix the flaky test. I mocked fetching the course data and seat data since it can ocassionally take a long time to do that and therefore the `seat_type` input element wouldn't be filled with seat types. Because of that I mocked the function so that the input element is filled from within the test.

Can you please take a look?

File: ecommerce/static/js/test/specs/views/coupon_create_view_spec.js [hunk 1]
```diff
@@ -55,28 +28,30 @@ define([
...
                 view.$el.find('[name=title]').val('Test Enrollment').trigger('change');
                 view.$el.find('[name=code_type]').val('enrollment').trigger('change');
                 view.$el.find('[name=client_username]').val('test_client').trigger('change');
                 view.$el.find('[name=start_date]').val('2015-01-01T00:00').trigger('change');
                 view.$el.find('[name=end_date]').val('2016-01-01T00:00').trigger('change');
                 view.$el.find('[name=price]').val('100').trigger('change');
-                spyOn($, 'ajax').and.callFake(function (options) {
-                    options.success(courseData);
+                spyOn(view.formView, 'fillFromCourse').and.callFake(function () {
+                    var seatTypes = [ $('<option></option>')
```
Comment (rlucioni) [inline ecommerce/static/js/test/specs/views/coupon_create_view_spec.js]: Nitpick: extra whitespace between the `[` and `$`.

**Comment by rlucioni** [comment]
:+1: , thanks for fixing this.

**Comment by marjev** [comment]
@rlucioni thanks for reviewing this! I addressed the nitpick.

Will your thumbs-up alone be enough for this to get merged?

**Comment by rlucioni** [comment]
@marjev yes, you're good to go once tests pass.

**Pull request merged**